docs/manpages/wforce.conf.5.md

```diff
@@ -427,19 +427,19 @@ cannot be called inside the allow/report/reset functions:
   "{login}" will be substituted for the actual IP address and login name. 
   For example:
 
-        setBlackistIPRetMsg("Go away your IP {ip} is blacklisted")
+        setBlacklistIPRetMsg("Go away your IP {ip} is blacklisted")
```

Comment:
these examples don't match the code function, which is unfortunate. I'm assuming that the docs were always wrong (I didn't check, I could, I'm in a hurry, sorry).

Reply:
The docs were always wrong yes